scripts/install_clang.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -ex
+
+CLANG_VERSION="$1"
+
+git clone https://github.com/llvm/llvm-project.git
+cd llvm-project
+
+if [ "$CLANG_VERSION" != "latest" ]; then
+    git checkout tags/llvmorg-${CLANG_VERSION} -b llvmorg-${CLANG_VERSION}
+fi
+
+mkdir build
+cd build
+cmake -DLLVM_ENABLE_PROJECTS=clang -G "Unix Makefiles" ../llvm
```

Comment:
It's a bit odd IMO, but the default for LLVM/Clang is actually Debug mode so think this needs a "-DCMAKE_BUILD_TYPE=Release" flag